src/pages/lists/multiselect-list/index.tsx

```diff
@@ -1,123 +1,406 @@
+const days = ['Today', 'Yesterday'];
+
+const createItem = (index: number, name: string, day: string): ListItemType => ({
+    id: index,
+    name: name,
+    checked: false,
+    day: day,
+});
+
+const generatedList: ListItemType[] = [];
+
+for (let i = 0; i < 5; i++) {
+    if (i < 3) {
+        generatedList.push(createItem(i, category[i], 'Today'));
+    } else {
+        generatedList.push(createItem(i, category[i - 3], 'Yesterday'));
+    }
+}
+const categorizeList = (list: ListItemType[]): any =>
+    list.reduce((r, a) => {
+        r[a.day] = r[a.day] || [];
+        r[a.day].push(a);
+        return r;
+    }, Object.create(null));
+
 const useStyles = makeStyles((theme: Theme) =>
     createStyles({
-        snackbar: {
-            [theme.breakpoints.up('md')]: {
-                left: `calc((100vw - ${DRAWER_WIDTH}px)/2 + ${DRAWER_WIDTH}px);`,
+        appbarRoot: {
+            padding: 0,
+        },
+        listItemRoot: {
+            backgroundColor: 'rgba(0, 123, 193, 0.05)',
+        },
+        card: {
+            marginBottom: `${theme.spacing(3)}px`,
+            [theme.breakpoints.down('sm')]: {
+                boxShadow: 'none',
+                borderRadius: 0,
+                marginBottom: `${theme.spacing(2)}px`,
+            },
+        },
+        cardContent: {
+            padding: 0,
+            '&:last-child': {
+                paddingBottom: 0,
+            },
+        },
+        checkboxIndeterminate: {
+            color: theme.palette.primary.main,
+        },
+        deleteBtn: {
+            backgroundColor: theme.palette.error.main,
+            color: colors.white[50],
+            height: '36px',
+            '&:hover': {
+                backgroundColor: colors.red[300],
             },
         },
-        emptyStateContainer: {
+        deleteRow: {
             display: 'flex',
-            flexDirection: 'column',
-            height: `calc(100vh - ${theme.spacing(8)}px)`,
+            justifyContent: 'flex-end',
+            alignItems: 'center',
+            marginBottom: `${theme.spacing(3)}px`,
         },
-        appbarRoot: {
-            padding: 0,
+        exampleContainer: {
+            padding: `${theme.spacing(3)}px`,
+            margin: '0 auto',
+            maxWidth: '816px',
+            [theme.breakpoints.down('sm')]: {
+                padding: 0,
+                boxShadow: 'none',
+                borderRadius: 0,
+                maxWidth: 'unset',
+            },
+        },
+        listItemIcon: {
+            marginLeft: `-${theme.spacing(1)}px`,
+        },
+        noResultListItem: {
+            marginLeft: `${theme.spacing(0.5)}px`,
+        },
+        panelHeaderRoot1: {
+            paddingLeft: `${theme.spacing(1)}px`,
+        },
+        panelHeaderRoot2: {
+            paddingLeft: `${theme.spacing(2)}px`,
+        },
+        panelHeaderTitle: {
+            color: colors.blue[500],
```

Comment:
use theme color?

Reply:
Removed this style, since using color='primary'.
https://github.com/pxblue/react-design-patterns/pull/112/files#diff-3d190fb5128299f650c1f4446a95686fd319b7924971542e3a878a5602c91a83R262